inferentia/scripts/setup.sh

```diff
@@ -64,11 +67,30 @@ for OPTS; do
         echo "Inferentia path set to ${INFRENTIA_PATH}"
         shift 2
         ;;
+        -t|--use-tensorflow)
+        USE_TENSORFLOW=0
+        echo "Installing tensorflow-neuron"
+        shift 1
+        ;;
+        -p|--use-pytorch)
+        USE_PYTORCH=0
```

Comment:
Why using 0 as the value to indicate using pytorch runtime? Usually we stick with the global truth values of 0 meaning false and 1 meaning true..

Same for USE_TENSORFLOW above.

Reply:
I was following the bash success/no success where anything but 0 means no-success. I'll change it

Reply:
Those are for exit codes for the functions..For truth values it is better to stick with 0(false) and 1(true)